subgraphs/reviews/reviews.graphql

```diff
@@ -0,0 +1,36 @@
+schema @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@key", "@override", "@shareable"]) {
+  query: Query
+}
+
+type Product implements ProductItf @key(fields: "id") {
+  id: ID!
+  reviewsCount: Int!
+  reviewsScore: Float! @shareable @override(from: "products")
+  reviews: [Review!]!
+}
+
+interface ProductItf {
+  id: ID!
+  reviewsCount: Int!
+  reviewsScore: Float!
+  reviews: [Review!]!
+}
+
+type Query {
+  _service: _Service!
+  _entities(representations: [_Any!]!): [_Entity]!
+  Hi: String!
```

Comment:
could we swap `Hi` with something that fits the Reviews domain, something like this?
```
review(id: ID!): Review
```

which would be similar to the products subgraph:
```
product(id: ID!): ProductItf
```